eureka/templates/base.html

```diff
@@ -73,14 +73,14 @@
                         <img src="{{ STATIC_URL }}img/logo-music.svg" alt="Columbia University, Department of Music"/>
                     </a>
                 </div>
-                <div class="col-md-4">
-                    {# Spacer #}
-                </div>
                 <div class="col-md-4">
                     <a href="https://ctl.columbia.edu">
                         <img src="{{ STATIC_URL }}img/ctl-logo-white.svg" alt="Center for Teaching and Learning, Columbia University"/>
                     </a>
+                </div>
+                <div class="col-md-4">
                     <a id="footer-login-link" href="/accounts/login/">Login</a>
```

Comment:
The login placement directly above Prof. Susser's name is a bit odd. Maybe find another spot for it?